docs/snippets/bitwarden-secrets-manager-secret-store.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   provider:
     bitwardensecretsmanager:
-      apiURL: https://vault.bitwarden.com
+      apiURL: https://api.bitwarden.com
       identityURL: https://identity.bitwarden.com
```

Comment:
nit: Worth adding comment, that the URLs can't include a trailing slashes, as this will cause 404 errors.

Reply:
Do you think it would be ok to trim them automatically?